release-notes/opensearch.release-notes-2.0.1.md

```diff
@@ -0,0 +1,12 @@
+## 2022-06-15 Version 2.0.1 Release Notes
+
+### Bug Fixes
+* [Type removal] Ignore _type field in bulk request ([#3504](https://github.com/opensearch-project/OpenSearch/pull/3504))
+* [Type removal] Revert removal of typed end-points for bulk, search, index APIs ([#3528](https://github.com/opensearch-project/OpenSearch/pull/3528))
+* [Type removal] Revert beat _type changes ([#3559](https://github.com/opensearch-project/OpenSearch/pull/3559))
```

Comment:
These three commits can be removed. First two commit added typed end-points (and type support in bulk request), the third commit reverts the first two.

Reply:
Updated the PR